### PR TITLE
Add exifread recipe

### PR DESCRIPTION
**Pull request by goanpeca**

### CODE REVIEW

File: recipes/exifread/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+test:
+  imports:
+    - exifread
+  commands:
+    - EXIF.py --help
+
+about:
+  home: https://github.com/ianare/exif-py
+  license: BSD
```
Comment (jakirkham) [inline recipes/exifread/meta.yaml]: Please add number of clauses (e.g. `BSD 3-Clause`).

File: recipes/exifread/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
+{% set pypi_name = "ExifRead" %}
+{% set name = pypi_name|lower %}
```
Comment (jakirkham) [inline recipes/exifread/meta.yaml]: Hmm...seems like it should be the same from looking at the links.

Comment (jakirkham) [inline recipes/exifread/meta.yaml]: Just tried the URL below and it worked for me. 😕 

https://pypi.io/packages/source/E/ExifRead/ExifRead-2.1.2.tar.gz

Though have had annoyances with these URLs before. Hence the way the [example]( https://github.com/conda-forge/staged-recipes/blob/1abe1dc23337e07188117e6db77014321f8677a0/recipes/example/meta.yaml ) is now. Maybe worth giving that a try.

**Comment by jakirkham** [comment]
While they don't list Python 3.5 support, I'm having trouble thinking of a way that we would run into issues as they already support Python 3.4 and it is a Python only package. This is probably just a sign they haven't updated their CI recently.

**Comment by jakirkham** [comment]
LGTM

**Comment by goanpeca** [comment]
Merge?

**Comment by jakirkham** [comment]
Sorry lost this one in the noise. 😄

**Comment by jakirkham** [comment]
Travis CI is being a bit slow. So it might take a bit for this to churn through. Just so you know.

**Pull request merged**